Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN chown -R app .
 
 USER app
 
-ENV JAVA_OPTS="-XX:+UseParNewGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10"
```

Comment:
Yea. I do feel like there should be a button in this new review system where we can mark a thread as resolved and collapse it